src/api/auth/auth.service.ts

```diff
@@ -0,0 +1,23 @@
+import { JwtService } from '@nestjs/jwt';
+import { Injectable } from '@nestjs/common';
+
+interface User {
+  password: string;
+  name: string;
+  email: string;
+}
+
+@Injectable()
+export class AuthService {
+  constructor(private readonly jwtService: JwtService) {}
+
+  login(user: User) {
+    const payload = { ...user };
+    return {
+      accessToken: this.jwtService.sign(payload, {
```

Comment:
이러면 payload 를 별도의 변수로 뺀 의미가 없어보여요! 그냥 아래처럼 해도 괜찮을 것 같습니다

```suggestion
    return {
      accessToken: this.jwtService.sign(user, {
```

Reply:
이친구를 저 위에 과정을 거치지 않고 바로 user에 넣어버리면 
Error: Expected "payload" to be a plain object.
에러가 뜨더라구요..
일단은 payload로 넣겠습니다!

Reply:
...?! 그건 뭐지 🤔 일단 고고하고 나중에 함 같이 보시졍